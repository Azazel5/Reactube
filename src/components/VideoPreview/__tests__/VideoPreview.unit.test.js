import React from 'react'
import {shallow} from 'enzyme'
import VideoPreview from '../VideoPreview'

describe('<VideoPreview />', () => {
    test('renders <VideoPreview />', () => {
        const wrapper = shallow(<VideoPreview />)
        expect(wrapper).toMatchSnapshot()
    })

    test('renders <VideoPreview /> horizontally', () => {
        const wrapper = shallow(<VideoPreview horizontal/>)
        expect(wrapper).toMatchSnapshot() 
    })

})