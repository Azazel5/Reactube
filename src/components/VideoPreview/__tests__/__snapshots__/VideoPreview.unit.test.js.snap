// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VideoPreview /> renders <VideoPreview /> 1`] = `<div />`;

exports[`<VideoPreview /> renders <VideoPreview /> horizontally 1`] = `<div />`;
